scrapers/utils/__init__.py

```diff
@@ -1,15 +1,19 @@
-import re
+import regex
 
 from .lxmlize import LXMLMixin  # noqa
 from .lxmlize import url_xpath  # noqa
 
+_phone_pattern = regex.compile(r"\(?\d{3}\)?\s?-?\d{3}-?\d{4}")
```

Comment:
Moving the `.compile` out of the functions should make them appreciably faster as we're no longer re-compiling on every run.